uaclient/mainwindow.py

```diff
@@ -49,6 +50,11 @@
 
 
 class DataChangeHandler(QObject):
+
+    def __init__(self, node_signal_dict):
+        super(DataChangeHandler, self).__init__()
```

Comment:
A `super` call like this is a smell for me, but this seems odd for other reasons, too: this class isn't inheriting from `DataChangeHandler`. Is this a typo, or leftovers from an earlier version?